### PR TITLE
Change dialog title, from 'App.Yaml' to 'app.yaml'

### LINKED ISSUE #597: Browse for app.yaml location dialog title has strange capitalization. 'Browse for Your App.Yaml Location'
appyaml

### PR DESCRIPTION
**Pull request by chanseokoh**
Fixes #597.

Checked it works.

### CODE REVIEW

File: google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties [hunk 1]
```diff
@@ -22,7 +22,7 @@ appengine.cloudsdk.location.label=Cloud SDK directory\:
 appengine.cloudsdk.location.browse.window.title=Browse to Cloud SDK directory
 appengine.cloudsdk.location.missing.message=Please select a Cloud SDK home directory.
 appengine.cloudsdk.project.missing.message=Please select a project.
-appengine.appyaml.location.browse.window.title=Browse for Your App.Yaml Location
+appengine.appyaml.location.browse.window.title=Browse for Your app.yaml Location
```
Comment (patflynn) [inline google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties]: ..**y**our..

Comment (chanseokoh) [inline google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties]: Fixed. Hope 'Location' is fine.

Comment (chanseokoh) [inline google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties]: Fixed 'location' too.

Comment (etanshaul) [inline google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties]: maybe its must me, but browse "for" your app.yaml location sounds awkward. Maybe "to" instead.

Comment (patflynn) [inline google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties]: yeah I agree. 

Change to 'to' and then LGTM

**Comment by chanseokoh** [comment]
For consistency, I also changed its counterpart to 'Browse to your Dockerfile location'.

**Comment by etanshaul** [comment]
lgtm

**Comment by patflynn** [comment]
Go ahead and squash and merge this.

**Pull request merged**